_docs/guides/integrating-vms.md

```diff
@@ -68,18 +75,19 @@ On the DB machine:
   kubectl delete service mysql
   ```
 
-  Run istioctl to configure the service (on your admin machine):
-
-  ```bash
-  istioctl register mysql PORT IP
-  ```
-
-  Note that the 'db' machine does not need and should not have special kubernetes priviledges.
+  Note that the 'mysqldb' virtual machine does not need and should not have special Kubernetes privileges.
 
 ## Using the mysql service
 
-The ratings service in bookinfo will use the DB on the machine. To verify it works, you can
-modify the ratings value on the database.
+The ratings service in bookinfo will use the DB on the machine. To verify that it works, create version 2 of the ratings service that uses the mysql db on the VM. Then specify route rules that force the review service to use the ratings version 2.
+```bash
+# Create the version of ratings service that will use mysql back end
+kubectl create -f samples/bookinfo/kube/bookinfo-mysql.yaml
+
+# Create route rules that will force bookinfo to use the ratings back end
+istioctl create -f samples/bookinfo/kube/route-rule-ratings-db.yaml
```

Comment:
I think you want the -mysql versions here

Reply:
bookinfo-mysql.yaml will create a version of the ratings service that is backed my mysql-db.
the route rule 'route-rule-ratings-db.yaml' will send all ratings svc traffic to that version of ratings service.
The 'route-rule-ratings-db' is common to both mongodb and mysqldb.